basics/imports-and-modules.md

```diff
@@ -0,0 +1,57 @@
+# Import ve Modül Sistemi
+
+{{#img-right}}turtle.svg{{/img-right}}
+
+D dilinin ana tasarım amacı tutarlı olmak ve uç durumlardan uzak durmaktır.
+Bu durum [_turtles all the way down_](https://en.wikipedia.org/wiki/Turtles_all_the_way_down) olarak söylenir.
```

Comment:
"Turtles all the way down" bunu nasıl çevireceğimiz konusunda hiçbir fikrim yok

Reply:
Doğrusunu istersen bende bilmiyorum. Ali'den yardım isteyebiliriz belki. 

Çeviri ve ingilizce konusunda iyi değilim ama senin tek başına da olsa bu işe kalkıştığını görünce seni tek başına bırakamazdım :+1:

Reply:
Kesinlikle katılıyorum Ali hocam bu konuda en doğru kaynak olacaktır.

Ben şu an staj yapıyorum ve staj yerinde işim bitti ve 3 hafta sürem var. Şu an yatıyorum sadece vaktimi bu şekilde değerlendireyim istedim ancak oldukça zor geliyor çeviri. Okuduğumu anlamakta zorlanmam, terimlere hakim olamamak, Türkçe eksikliği zorlaştırıyor. Ama bu mantıkta her gün çeviri yapıp sizinle üzerinden geçebilirsek aslında oldukça yol kat edebiliriz diye düşünüyorum. Böyle bir yardım beklemiyordum açıkçası :) Teşekkür ederim 👍

Reply:
Eğer zorluklarla karşılaşıyorsan doğru yoldasın demektir :)

Rica ederim. Elimden geldiğince yardımcı olmaya çalışıyorum. Amaç güzel bir şeyler üretmek olunca yardım etmekte keyif veriyor.